lib/db/set.go

```diff
@@ -324,10 +324,7 @@ func DropFolder(db *Instance, folder string) {
 	db.dropFolder([]byte(folder))
 	db.dropMtimes([]byte(folder))
 	db.dropFolderMeta([]byte(folder))
-	bm := &BlockMap{
-		db:     db,
-		folder: db.folderIdx.ID([]byte(folder)),
-	}
+	bm := NewBlockMap(db.Lowlevel, db.folderIdx.ID([]byte(folder)))
```

Comment:
Just `NewBlockMap(db.Lowlevel)` and then do the `folderIdx` stuff there, as it belongs to `Lowlevel`.